recipes/libipt/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(libipt REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_compile_features(test_package PRIVATE cxx_std_11 cxx_constexpr)
+target_link_libraries(${PROJECT_NAME} libipt::libipt)
```

Comment:
I wonder how did it work here in Linux gcc/clang without last addition from you, @uilianries

Reply:
@miklelappo the set_property is for CMakeDeps generator, as we are not using here, we didn't see. But CMakeDeps should the default for Conan 2.0, so consider it as broken in the future 😅

Reply:
that's hard